src/cache.ml

```diff
@@ -8,12 +8,11 @@ module Make(IO : S.IO)(Client : S.Client with module IO = IO)(Params : S.Cache_p
   let set r key data =
     let key = Params.cache_key key in
     let data = Params.string_of_data data in
```

Comment:
Ideally, I would have this function return `bool IO.t` but don't feel like changing the signature